packages/@okta/vuepress-site/docs/guides/sign-into-web-app-redirect/main/go/createproject.md

```diff
@@ -0,0 +1,13 @@
+1. Make sure that you have [Go installed](https://go.dev/dl/).
+
+2. Create a new directory to store your project inside, and then `cd` inside it.
+
+3. Initialize your module using `go mod init`, for example:
+
+```shell
+go mod init okta/go-quickstart
+```
+
+4. Create a simple starter structure for your project. Have a look at our [sample structure](https://github.com/okta-samples/okta-go-gin-sample).
+
+> **Note**: This guide uses Golang version 1.17.6 and Gin 1.7.7.
```

Comment:
the note about using CLI to generate the project is missing now

Reply:
Again, really weird! I've added it back in.